chrome/views/gmailView.js

```diff
@@ -25,12 +25,13 @@ GmailToTrello.GmailView = function(parent) {
         emailBody: '.adP:first', // Was: '.adO:first'
         emailAttachments: '.aZo', // Was: '.aQy',
         emailThreadID: '.a3s.aXjCH',
-        viewport: '.aeJ:first',
+        viewport: '[role="main"]',
         viewportSplit: '.aNW:first', //reading panel
         expandedEmails: '.h7',
         hiddenEmails: '.kv',
         emailInThreads: '.kv,.h7',
         timestamp: '.gH .gK .g3:first',
+        messageTimes: '.g3',
```

Comment:
This is new. Used to get all of the times of all the messages in the thread, so we can use them for the data comparison in loadThreadTrelloCards.